railties/lib/rails/application/bootstrap.rb

```diff
@@ -20,13 +20,13 @@ module Bootstrap
       initializer :set_eager_load, group: :all do
         if config.eager_load.nil?
           warn <<-INFO
```

Comment:
Just enough changing one line here to fix the output.

```suggestion
          warn <<~INFO
```

Reply:
I know, I mentioned that in the opening comment. There are many more instances where we use `<<-` so I'd like to change all of those together.
I only reverted the original commit in this PR. Do you think I should change all places to use `<<~` in this PR itself?

Reply:
IMO no. The purpose of #36214 is addressing mismatched ending heredoc indentation warning, broken output is a side-effect.

In the past, I was looking for a systematic way that detect that mismatched ending heredoc indentation. But AFAICS enabling `Layout/IndentHeredoc` cop is an only way.

`Layout/IndentHeredoc` cop can detect and auto-correct that that mismatched ending heredoc indentation, but it also enforce to indent the content, so I've leave as it is for now to keep git history, and fix one by one when a file which contain that mismatched ending heredoc indentation (e.g. https://github.com/rails/rails/commit/5d858b5d3e5539ede0ea541a8e14126a3ee30800#diff-0bf64de97e08896fd543f61492174b88L15).

https://github.com/kamipo/rails/commit/99d6e292f0412f641d6de0202539499b34e96cb2

Replacing `<<-` to `<<~` by hand is not the right time yet to me.

Reply:
Ok. Thanks for explaining. I'll change the PR to use `<<~` for now.